CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.3)
 PROJECT(lmms)
 
 SET(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules" ${CMAKE_MODULE_PATH})
+set(CMAKE_INSTALL_PREFIX ${CMAKE_BINARY_DIR})
```

Comment:
@Reflexe It looks weird. Are there any reasons for it?

Reply:
@PhysSong probably left it by mistake after testing. thanks.